clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1445,3 +1445,6 @@ def ReadOnlyPlacementChecks : DiagGroup<"read-only-types">;
 
 // Warnings and fixes to support the "safe buffers" programming model.
 def UnsafeBufferUsage : DiagGroup<"unsafe-buffer-usage">;
+
+// Warnings for fp accuracy already set.
+def FPAccuracyAlreadySet : DiagGroup<"fp-accuracy-value">;
```

Comment:
This can be removed and you can use an ad hoc group in the definition of `warn_function_fp_accuracy_already_set`.

Reply:
What do you mean but an "ad hoc" group?

Reply:
Sorry, I should have spelled that out. :-D

```
def warn_foo : Warning<
  "foo is awful, how could you do such a thing?",
  InGroup<DiagGroup<"foo-is-the-worst">>;
```
this makes a one-off diagnostic group `-Wfoo-is-the-worst` and associates just this one warning with the group. It works great for times when there's only one diagnostic under the group flag.